content/en/docs/instrumentation/go/getting-started.md

```diff
@@ -25,18 +25,27 @@ directory.
 ```go
 package main
 
-// Fibonacci returns the n-th fibonacci number.
-func Fibonacci(n uint) (uint64, error) {
-	if n <= 1 {
-		return uint64(n), nil
-	}
+import (
+        "errors"
+)
 
-	var n2, n1 uint64 = 0, 1
-	for i := uint(2); i < n; i++ {
-		n2, n1 = n1, n1+n2
-	}
+var ErrOverflow = errors.New("unsigned integer overflow")
 
-	return n2 + n1, nil
+// Fibonacci returns the n-th fibonacci number.
+func Fibonacci(n uint) (uint64, error) {
+        if n <= 1 {
+                return uint64(n), nil
+        }
+
+        var n2, n1 uint64 = 0, 1
+        for i := uint(2); i < n; i++ {
+                if (n1 + n2) + n1 < n1 {
+                        return 0, ErrOverflow
+                }
+                n2, n1 = n1, n1+n2
+        }
+
+        return n2 + n1, nil
```

Comment:
Please use "tabs" instead of "spaces".

Reply:
Addressed with https://github.com/open-telemetry/opentelemetry.io/pull/2901/commits/806d4094e6a1fcf6ce681643dad0f86b534853e2